HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTrackstersCLUE3DHighL1Seeded_cfi.py

```diff
@@ -100,34 +94,38 @@
       2,
       2
     ),
-    eid_input_name = cms.string('input'),
-    eid_output_name_energy = cms.string('output/regressed_energy'),
-    eid_output_name_id = cms.string('output/id_probabilities'),
-    eid_min_cluster_energy = cms.double(1),
-    eid_n_layers = cms.int32(50),
-    eid_n_clusters = cms.int32(10),
     computeLocalTime = cms.bool(False),
     doPidCut = cms.bool(True),
     cutHadProb = cms.double(999.),
     type = cms.string('CLUE3D')
-
     ),
     pluginPatternRecognitionByFastJet = cms.PSet(
         algo_verbosity = cms.int32(0),
         antikt_radius = cms.double(0.09),
-        eid_input_name = cms.string('input'),
-        eid_min_cluster_energy = cms.double(1),
-        eid_n_clusters = cms.int32(10),
-        eid_n_layers = cms.int32(50),
-        eid_output_name_energy = cms.string('output/regressed_energy'),
-        eid_output_name_id = cms.string('output/id_probabilities'),
         minNumLayerCluster = cms.int32(5),
         type = cms.string('FastJet')
     ),
+    pluginInferenceAlgoTracksterInferenceByDNN = cms.PSet(
```

Comment:
I'm confused. I see that here you use the `pluginInferenceAlgoTracksterInferenceByDNN` with a TICLv4 model, while in the `TracksterProducer` and the linking variant, there is also the plugin `TracksterInferenceByCNNv4`. If we configure the `DNN` one, should we expect regression?

Reply:
Hi Marco, so this additional empty plugin is set up in anticipation of future models that might arrive later, so we prepared the configuration for it in advance. Thanks!